public_html/index.html

```diff
@@ -8,6 +8,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <title>Mozilla – The Future of Web Games</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="@mozilla">
```

Comment:
I think this should read `games.mozilla.org` - what’s it look like in the Twitter Card Previewer?

Reply:
I changed, but in previewer needs the og meta tags to look something